bin/is-website-vulnerable.js

```diff
@@ -1,48 +1,35 @@
 #!/usr/bin/env node
 /* eslint-disable no-process-exit */
-'use strict'
-
+/* eslint no-console: ["error", { allow: ["error", "log"] }] */
 const os = require('os')
 const debug = require('debug')('is-website-vulnerable')
 const argv = require('yargs').argv
-const { prompt } = require('enquirer')
-const isUrl = require('is-url-superb')
 const { Audit, RenderConsole, RenderJson, Utils } = require('../index')
+const getUrl = require('./get-url')
 
-let url = process.argv[2]
-debug(`received url argument: ${url}`)
-
-const isWindows = os.type() === 'Windows_NT'
-const isJson = !!argv.json
-const showProgressBar = !isJson && !isWindows
+const getStatus = () => {
+  const isWindows = os.type() === 'Windows_NT'
+  const isJson = !!argv.json
+  const showProgressBar = !isJson && !isWindows
 
-;(async () => {
-  try {
-    if (!url) {
-      if (isJson) {
-        throw new Error('Please provide a URL of a website to scan')
-      } else {
-        console.error('Woops! You forgot to provide a URL of a website to scan.')
+  return { isWindows, isJson, showProgressBar }
+}
 
-        try {
-          const response = await prompt({
-            type: 'input',
-            name: 'url',
-            message: 'Please provide a URL to scan:',
-            validate: input => input && input.length > 0 && isUrl(input)
-          })
-          url = response.url
-        } catch (_) {
-          throw new Error('URL input prompt failed')
-        }
-      }
-    }
+const getOptions = () => {
```

Comment:
Same as above, we can probably name it in a more descriptive way, how about `getLighthouseOptions` ?

Reply:
Ok.